.github/workflows/platform.dependencies.yml

```
@@ -81,6 +84,15 @@ jobs:
           subscriptionId: '${{ secrets.ARM_SUBSCRIPTION_ID }}'
           managementGroupId: '${{ secrets.ARM_MGMTGROUP_ID }}'
           removeDeployment: '${{ env.removeDeployment }}'
+      - name: Set msi principal ID output
+        id: print_msi_prinId
+        uses: azure/powershell@v1
+        with:
+          inlineScript: |
+            $deploymentOutput = '${{ steps.deploy_msi.outputs.deploymentOutput }}'
+            $msiPrincipalId = (ConvertFrom-Json $deploymentOutput).msiPrincipalId
+            Write-Output ('::set-output name=msiPrincipalId::{0}' -f $msiPrincipalId)
```

```suggestion
            Write-Output ('::set-output name={0}::{1}' -f 'msiPrincipalId', $msiPrincipalId)
```
Very minor - but is consistent with ADO pipeline